src/SerilogTracing.Sinks.OpenTelemetry/OpenTelemetryLoggerConfigurationExtensions.cs

```diff
@@ -0,0 +1,182 @@
+            options.TracesEndpoint = tracesEndpoint;
+            options.Protocol = protocol;
+            headers?.AddTo(options.Headers);
+            resourceAttributes?.AddTo(options.ResourceAttributes);
+        });
+    }
+    
+    /// <summary>
+    /// Audit to an OTLP exporter, waiting for each event to be acknowledged, and propagating errors to the caller.
+    /// </summary>
+    /// <param name="loggerAuditSinkConfiguration">
+    /// The `AuditTo` configuration object.
+    /// </param>
+    /// <param name="configure">The configuration callback.</param>
+    public static LoggerConfiguration OpenTelemetry(
+        this LoggerAuditSinkConfiguration loggerAuditSinkConfiguration,
+        Action<OpenTelemetrySinkOptions> configure)
+    {
+        if (configure == null) throw new ArgumentNullException(nameof(configure));
+
+        var options = new OpenTelemetrySinkOptions();
+        
+        configure(options);
+
+        var exporter = Exporter.Create(
+            logsEndpoint: options.LogsEndpoint,
+            tracesEndpoint: options.TracesEndpoint,
+            protocol: options.Protocol,
+            headers: new Dictionary<string, string>(options.Headers),
+            httpMessageHandler: options.HttpMessageHandler);
+
+        var sink = new OpenTelemetrySink(
+            exporter: exporter,
+            formatProvider: options.FormatProvider,
+            resourceAttributes: new Dictionary<string, object>(options.ResourceAttributes),
+            includedData: options.IncludedData);
+
+        return loggerAuditSinkConfiguration.Sink(sink, options.RestrictedToMinimumLevel, options.LevelSwitch);
+    }
+    
+    /// <summary>
+    /// Audit to an OTLP exporter, waiting for each event to be acknowledged, and propagating errors to the caller.
+    /// </summary>
+    /// <param name="loggerAuditSinkConfiguration">
+    /// The `AuditTo` configuration object.
+    /// </param>
+    /// <param name="logsEndpoint">
+    /// The full URL of the OTLP logs exporter endpoint.
+    /// </param>
+    /// <param name="tracesEndpoint">
+    /// The full URL of the OTLP traces exporter endpoint.
+    /// </param>
+    /// <param name="protocol">
+    /// The OTLP protocol to use.
+    /// </param>
+    /// <param name="headers">
+    /// Headers to send with network requests.
+    /// </param>
+    /// <param name="resourceAttributes">
+    /// A attributes of the resource attached to the logs generated by the sink. The values must be simple primitive
+    /// values: integers, doubles, strings, or booleans. Other values will be silently ignored.
+    /// </param>
+    /// <returns>Logger configuration, allowing configuration to continue.</returns>
+    public static LoggerConfiguration OpenTelemetry(
+        this LoggerAuditSinkConfiguration loggerAuditSinkConfiguration,
+        string logsEndpoint = OpenTelemetrySinkOptions.DefaultEndpoint,
+        string tracesEndpoint = OpenTelemetrySinkOptions.DefaultEndpoint,
```

Comment:
Perhaps `logsEndpoint` and `tracesEndpoint` could both be nullable, so that when the logs and traces are going to different destinations (with potentially different protocols and auth headers) two different sinks instances could be configured?

E.g.

```csharp
    .WriteTo.OpenTelemetry(logsEndpoint: "https://amazinglogs.example.com", protocol: OtlpProtocol.HttpProtobuf)
    .WriteTo.OpenTelemetry(tracesEndpoint: "https://unrealtraces.example.com", protocol: OtlpProtocol.Grpc)
```

(And accepting that null logs endpoint + null traces endpoint = silent sink..)

Reply:
Also for the scenario where you want one or the other